### ISSUE TITLE
Add support for implementing commands in the new CLI alone

### ISSUE THREAD

**Issue by akalipetis** [platformsh/cli#78]
Currently, we have only implemented `project:init` command in the new CLI. This command was separate and did not need interoperability with the Legacy CLI, in terms of authentication and configuration.

We should implement that foundation, in order to be able to work on new commands in the new CLI.

```[tasklist]
### Tasks

- [ ] Add Legacy CLI authentication support
- [ ] Add Legacy CLI configuration support
```

**Comment by dman-coders** [comment]
I'm guessing that the difference between legacy and only-new commands like this is also what causes the "did you mean" suggestion to be unaware of the new command(s) also?

https://platformsh.slack.com/archives/C05RZU4ES8G/p1723439099274829

```
upsun init

[CommandNotFoundException]      
Command "init" is not defined.  
Did you mean one of these?      
environment:init            
int:act                     
```

Is there anyway these commands can be indexed in a unified way, or is that a lot of work for small gain?
^ does not even suggest `project:init`, which is what I expected as the default.

**Comment by akalipetis** [comment]
Yes, that's the reason this does not work, although `init` would not work even after this has been implements - you can use `ify` as a shortcut if you want.
